src/test/xxd/xxd.c

```diff
@@ -0,0 +1,897 @@
+/* xxd: my hexdump facility. jw
+ *
+ *  2.10.90 changed to word output
+ *  3.03.93 new indent style, dumb bug inserted and fixed.
+ *	    -c option, mls
+ * 26.04.94 better option parser, -ps, -l, -s added.
+ *  1.07.94 -r badly needs - as input file.  Per default autoskip over
```

Comment:
Why are we including a custom xxd? This would be your question normally... :)

Reply:
`-n` options is recent and not included in current ubuntu (which CI uses)

Reply:
Makes sense.